data/lang/equipment-core/en.json

```diff
@@ -326,5 +326,9 @@
    "UNOCCUPIED_CABIN_DESCRIPTION" : {
       "description" : "",
       "message" : "Required for transport of a single passenger"
-   }
+   },
+   "ERROR_LANDING_GEAR_DOWN": {
+      "description": "",
+      "message": "ERROR: Landing gear is down!"
```

Comment:
Maybe we could change this to something along the lines of: "Cannot initiate hyperjump while landing gear is down".